packages/generator/generators/module-generator/templates/ts-template/tsconfig.json.handlebars

```diff
@@ -1,13 +1,23 @@
 {
-  "extends": "@modern-js/tsconfig/base",
-  "declaration": true,
   "compilerOptions": {
-    "baseUrl": "./",
+    "allowJs": true,
```

Comment:
Should we remove @modern-js/tsconfig from dependencies

Reply:
yes, we can remove it